pallets/valve/src/weights.rs

```diff
@@ -0,0 +1,105 @@
+
+//! Autogenerated weights for `pallet_valve`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-03-02, STEPS: `20`, REPEAT: 64, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
how frequently do we have to regenerate these weights?

Reply:
Whenever we change something that could change the weights. That could be changing the logic in a function or changing pallet config params